.github/workflows/test-and-deploy.yml

```diff
@@ -0,0 +1,78 @@
+name: Test and Deploy
+on:
+  push:
+    branches: [ '*' ]
+    tags: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        python-version: [ '2.7', '3.5', '3.6', '3.7', '3.8', '3.9' ]
+    steps:
+      - name: Checkout sendgrid-python
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
This is not needed since the SendGrid helpers use Docker for testing. Also, need to set the `version` env var, similar to what was done in your Java PR.

Reply:
Done. Removed the set up python step.